src/main/java/fifthtask/LeapYear.java

```diff
@@ -0,0 +1,46 @@
+package fifthtask;
+
+import utils.InputOutputUtils;
+import java.io.IOException;
+import java.time.Year;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LeapYear {
+    private static final String INPUT_FILE = "src/main/resources/input_files/input_fifthtask";
+    private static final String OUTPUT_FILE = "target/logs/output_fifthtask";
+
+    public static void main(String[] args) throws IOException {
+        int number = InputOutputUtils.readOnlyOneIntFromFile(INPUT_FILE);
+//        InputOutputUtils.writeToFile(calcLeapYearUsingWhile(number), OUTPUT_FILE);
+        InputOutputUtils.writeToFile(calcLeapYearUsingForLoop(number), OUTPUT_FILE);
+
+    }
+
+    public static List<Integer> calcLeapYearUsingWhile(int n) {
+        List<Integer> years = new ArrayList<>();
+        int year = Year.now().getValue();
+        int count = 0;
+        while (count < n) {
+            year += 1;
```

Comment:
you can try to substitute **year += 1;** with **++year;** both in this method and the next one

Reply:
Will do. Thanks.